client/app/bundles/course/lesson-plan/pages/LessonPlanEdit/ItemRow/PublishedCell.jsx

```diff
@@ -8,20 +7,18 @@ const styles = {
   },
 };
 
-class PublishedCell extends React.PureComponent {
```

Comment:
Will this compromise the performance?

Reply:
You mean `PureComponent`? It can _potentially_ improve performance, since it kinda does memoisation on a shallow level, i.e. if the props passed in are the same between re-renders of the parent component, this component won't change.

Can see more at: https://reactjs.org/docs/react-api.html#reactpurecomponent

Reply:
Ah yea i know.. i mean here it seems that PureComponent is changed to just Component

Reply:
Ah ok, I'll look into it!